update/delta/delta_1.1.0.sql

```diff
@@ -309,8 +309,8 @@ DROP TRIGGER valve_node_set_type ON qwat_od.valve;
 DROP FUNCTION qwat_od.ft_valve_node_set_type();
 
 -- We need to manually remove those 3 triggers functions, as they cannot be removed automatically by the rewrite views process
-DROP FUNCTION qwat_od.ft_element_valve_delete();
-DROP FUNCTION qwat_od.ft_element_valve_insert();
-DROP FUNCTION qwat_od.ft_element_valve_update();
+DROP FUNCTION qwat_od.ft_element_valve_delete() cascade;
+DROP FUNCTION qwat_od.ft_element_valve_insert() cascade;
+DROP FUNCTION qwat_od.ft_element_valve_update() cascade;
```

Comment:
oh no let's don't do cascades...if there is anything associated outside of qwat, it is problematic.
better write down all need drops.